ProcessMaker/processmaker
# Implement a new login failed page

## Implement a new login failed page

## Solution
- Implement a new login failed page


https://github.com/ProcessMaker/processmaker/assets/8028650/3df3641f-d04b-4da2-a588-927072381f77


## How to Test
- Configure a SSO drive
- Setup a user with SSO
- Change the user status to inactive
- Try to login with the inactive user
- See the new login failed page

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-7394

## Code Review Checklist

ci:next
ci:deploy
ci:package-auth:bugfix/FOUR-7394B
